lib/ansible/module_utils/hcloud.py

```diff
@@ -0,0 +1,61 @@
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
```

Comment:
We tend to add this snippet at the top of every file (usually below the shebang if there is one).
```suggestion
# -*- coding: utf-8 -*-

# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
```